ibc-clients/ics07-tendermint/types/src/error.rs

```diff
@@ -19,6 +19,8 @@ use tendermint_light_client_verifier::Verdict;
 pub enum TendermintClientError {
     /// decoding error: `{0}`
     Decoding(DecodingError),
+    /// identifier error: `{0}`
+    Identifier(IdentifierError),
```

Comment:
Couldn't we utilize `Decoding(DecodingError::Identifier(e))` instead?

Reply:
I think I added this variant back in by mistake.